folks.yml

```diff
@@ -108,3 +108,8 @@
   username: tim.johnson
   github: timjohnson-valtech
   npm: valtech_tim
+
+- name: Rohan Vasishth
+  username: rohan.vasishth
+  github: rohanvasishth
+  npm: 
```

Comment:
Hey @serhalp or @risharma , @rohanvasishth mentioned that he doesn't have an `npm` user. And since we use an `NPM_TOKEN` to have access to the good eggs registry it didn't seem necessary to create an npm user. Is it ok to leave this field blank? Or should he create an npm user anyway and put it here?

Reply:
@bobzoller can we just remove these `npm` keys from this file? Also, bump: https://github.com/goodeggs/key-pository/issues/39.